src/types/index.ts

```diff
@@ -1,8 +1,38 @@
 type JSTypes = 'undefined' | 'object' | 'boolean' | 'number' | 'bigint' | 'string' | 'symbol' | 'function';
+export enum BracketTypes {
+  UNDEFINED = 'undefined',
+  OBJECT = 'object',
+  BOOLEAN = 'boolean',
+  NUMBER = 'number',
+  BIGINT = 'bigint',
+  STRING = 'string',
+  ARRAY = 'ARRAY',
```

Comment:
let's match the format of other enums.
```suggestion
  ARRAY = 'array',
```